File: Practice(Beginner)/C++/Gross Salary.cpp
// Source: https://www.codechef.com/problems/FLOW011

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t, salary = 0;
    cin >> t;
    while (t--){
        cin >> salary;
        if(salary < 1500){
            cout << fixed << setprecision(2) << 2.0 * salary << endl;
        }else{
            float hra = 500, da = salary * 0.98;
            cout << fixed << setprecision(2) << salary + hra + da << endl;
        }
    }
    return 0;
}

File: Practice(Beginner)/Python3/Gross_Salary.py
# Source: https://www.codechef.com/problems/FLOW011

n = int(input())
for i in range(n):
    bs = int(input())
    if(bs < 1500):
        hra = bs*0.10
        da = bs*0.90
    else:
        hra = 500
        da = bs*0.98
    print("{:0.2f}".format(bs+hra+da))
